src/core/lib/experiments/experiments.yaml

```diff
@@ -149,3 +149,11 @@
   owner: alishananda@google.com
   test_tags: []
   allow_in_fuzzing_config: false
+- name: chttp2_allow_config_override_keepalive_permit_without_calls
```

Comment:
Can we give this a shorter name? (there's some cost involved in longer names here)

Reply:
really? what cost?

Reply:
done